app/javascript/components/layout/CustomTable.js

```diff
@@ -219,6 +226,7 @@ class CustomTable extends React.Component {
                 return (
                   <tr
                     key={rowIndex}
+                    aria-label={this.getRowAriaLabel(rowIndex)}
```

Comment:
is it possible to do this check inline instead of breaking it out?

Reply:
Yeah I can do that!